index.html

```diff
@@ -1554,32 +1554,11 @@ <h4><code>timeContainer</code></h4>
             the computed begin time of its parent element,
             or for the <code>&lt;body&gt;</code> element, to time zero.</p>
         </section> <!-- timecontainer-section -->
-        <section id="framerate-section">
-          <h4><code>ttp:frameRate</code></h4>
-          <p>If the document contains any time expression that uses the <code>f</code> metric,
-            or any time expression that contains a frames component,
-            the <code>ttp:frameRate</code> attribute MUST be present on the <code>&lt;tt&gt;</code> element.</p>
-        </section> <!-- framerate-section -->
```

Comment:
I'm not sure we want to remove `f`. The only thing we want to remove is the syntax that looks like SMPTE time code.

Reply:
Seems like unnecessary complexity to support frame counts if no other frame based expressions are permitted. Why not use ticks if an arbitrary rate time counting mechanism is needed?

I've realised I am not sure if subframes have any bearing on frame count time expressions.

Reply:
I see from the issue that the proposal was to retain `f` units so I should have checked more carefully. Will fix when I can.